### DISCUSSION TITLE
New Client (Athena)

### DISCUSSION THREAD

**Discussion by karamokobaD** [async-aws/aws#1390] [Show and tell]
Hi everyone :wave:  

Sorry to bother, we use async-aws in ours project, and for a recent one we need the Athena Client, so due to that with add a new Client (Athena) following your recommendation for a contribution. Here our Pull Request: [1388](https://github.com/async-aws/aws/pull/1388) waiting the team review :pray:

We really hope that the new Client could be, validate and merge rapidly :pray::pray:. That will let us keep using async. Otherwise because of the project progress and using Athena client became more and more urgent we will have to change our library.

Any feedback will be great, accepted or rejected.

**Comment by stof** [comment]
Duplicate of https://github.com/async-aws/aws/discussions/1387
